### PR TITLE
Release 2.1.4

### PR DESCRIPTION
**Pull request by stel**
Release v 2.1.4 with cocoa 2.4.1

/cc @radu-tutueanu, @jpsim

### CODE REVIEW

**Comment by stel** [comment]
The build is pending a review on iTC, will merge after it's completed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 2.1.3 Release notes (2017-01-17)
```
Comment (jpsim) [inline CHANGELOG.md]: incorrect version number and release date

Comment (stel) [inline CHANGELOG.md]: Ah.. :( will fix, fortunately it's not related to binaries.

**Comment by stel** [comment]
The build has been approved, merging the PR.

**Pull request merged**